packages/teleport/src/Databases/ConnectDialog/ConnectDialog.tsx

```diff
@@ -26,26 +26,16 @@ import { DbProtocol } from 'shared/services/databases';
 
 import { AuthType } from 'teleport/services/user';
 import TextSelectCopy from 'teleport/components/TextSelectCopy';
+import { generateTshLoginCommand } from 'teleport/lib/util';
 
 export default function ConnectDialog({
   username,
+  clusterId,
```

Comment:
I'm still new to learning TypeScript, so I was wondering if it is alright to add `clusterId` here?
I added `clusterId` here since the `generateTshLoginCommand` requires it and it was already included in the `Props` type for the `ConnectDialog` component.